pkg/plugins/models.go

```diff
@@ -296,6 +296,23 @@ func (e *Error) WithMessage(m string) *Error {
 	return e
 }
 
+func (e Error) PublicMessage() string {
```

Comment:
I just want to check - these are all fatal error states which would leave the plugin unable to successfully run, including if the configuration allows for unsigned plugins etc?

Reply:
perfect - thanks!